Stripe/StripeiOSTests/PayWithLinkButtonSnapshotTests.swift

```diff
@@ -69,6 +69,13 @@ class PayWithLinkButtonSnapshotTests: STPSnapshotTestCase {
         verify(sut)
     }
 
+    func testAddThenRemoveAccount() {
+        let sut = makeSUT()
```

Comment:
What is a `sut`?

Reply:
"System Under Test": It's a standard term in our tests (since 2015!), but I don't know who introduced it.